fv3_cap.F90

```diff
@@ -56,30 +60,61 @@ module fv3atm_cap_mod
 !
 !-----------------------------------------------------------------------
 !
-
+  !> ???
   type(ESMF_GridComp)                         :: fcstComp
+
+  !> ???
   type(ESMF_State)                            :: fcstState
+
+  !> ???
   type(ESMF_FieldBundle), allocatable         :: fcstFB(:)
+
+  !> ???
   integer,dimension(:), allocatable           :: fcstPetList
+
+  !> ???
```

Comment:
All these variables are local (private) variables and are not accessible outside the module. Are you going to add doxygen the every local variable in every module and subroutine? I thought we are adding doxygen just to subroutine arguments and public module variables.

Reply:
Good question, I will investigate and get back to you.

We need to document all the module variables that doxygen will complain about if undocumented. This is so we can turn on warning-check and ensure that no undocumented code is committed to the repo.